website/content/docs/connect/config-entries/mesh.mdx

```diff
@@ -47,24 +48,102 @@ TransparentProxy {
 ```
 
 </Tab>
-</Tabs>
-
-## Available Fields
-
-- `Kind` - Must be set to `mesh`
+<Tab heading="Kubernetes YAML">
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: Mesh
+metadata:
+  name: mesh
+spec:
+  transparentProxy:
+    meshDestinationsOnly: true
+```
 
-- `Namespace` `(string: "default")` <EnterpriseAlert inline /> - Specifies the namespace the config entry will apply to.
-  Must be set to `default`
+</Tab>
+<Tab heading="Kubernetes YAML (Consul Enterprise)">
+
+**NOTE:** A `Mesh` resource can be created in any Kubernetes
+namespace but it will apply to proxies across **all** namespaces. Only one
+`Mesh` resource can exist in the cluster.
```

Comment:
Should the same be said about the enterprise hcl? That only one entry can exist / cluster?